lib/reports/subscriber_list_subscriber_count_report.rb

```diff
@@ -0,0 +1,31 @@
+class Reports::SubscriberListSubscriberCountReport
+  class BadDateError < StandardError; end
+
+  attr_reader :url, :active_on_date
+
+  def initialize(url, active_on_date = nil)
+    @url = url
+    @active_on_date = active_on_date || Time.zone.now
+  end
+
+  def call
+    list = SubscriberList.find_by_url(url)
+    if list
+      list.subscriptions
+          .active_on(formatted_date)
+          .count
```

Comment:
Could you format the count into a string? I think just having a number on a line is easy to miss or not understand.

I'm thinking something like
`"Subscriber list for #{url} had #{count} subscribers on #{formatted_date}."`